dll/cpl/appwiz/createlink.c

```diff
@@ -14,6 +14,7 @@
 #include <commctrl.h>
 #include <shellapi.h>
 #include <strsafe.h>
+#include <shlwapi_undoc.h>
```

Comment:
Which specific function call does require that include?

Reply:
It's for `PathFindOnPathExW`. Added a comment.